packages/bms/package.json

```diff
@@ -53,9 +49,6 @@
     "data-structure": "^1.2.0",
     "iconv-lite": "^0.4.24",
     "invariant": "^2.2.4",
-    "lodash.assign": "^4.2.0",
-    "lodash.map": "^4.6.0",
-    "lodash.uniq": "^4.5.0",
-    "lodash.values": "^4.3.0"
```

Comment:
Unfortunately, `lodash-es` is not compatible with Node due to Node not supporting `import` syntax, hence the unit tests failed.

However, I want to keep `bms` project compatible with Node.js, since it may be reused in Node.js projects such as `bms-renderer`, also `bemuse-indexer`, etc. That’s why I am cherry-picking individual lodash functions from npm, instead of, say, using `lodash-es`.

In the future, `lodash` will support both CommonJS and ESM runtimes in the same package, by utilizing `"main"` (pointing to CommonJS) and `"module"` (pointing to ESM) keys in `package.json`.

While other packages has been doing this already, `lodash` will not do it until v5.0.0 is released.

Refs:

- https://github.com/lodash/lodash/issues/3745#issuecomment-380532439
- https://github.com/lodash/lodash/wiki/Roadmap/56353d95cbae58e1559e8974c0c8c5b326818320

Reply:
Alright, I'll return it to regular lodash later today.